README.md

```diff
@@ -20,16 +20,51 @@ Once the binary file has been obtained, users will need to update their environm
 
 Example for Ubuntu:
 ```
-export PATH={BANDAGE_DIR}:$PATH
+export PATH=~/projects/Bandage:$PATH
 ```
 
 Example for MacOS:
 ```
-export PATH={BANDAGE_DIR}/Bandage.app/Contents/MacOS:$PATH
+export PATH=~/projects/Bandage/Bandage.app/Contents/MacOS:$PATH
 ```
 
 In order to run the pre-built binary files successfully, ensure that the binary file is kept in the same directory with all of the other files that came with it.
 
+
+### Installing metaMDBG (for Linux OS only)
+
+To enable `metaMDBG` into YEAT's workflow, users will need to create a conda environment, install, and build from the source. Once created, YEAT will activate the environment whenever `metaMDBG` is called.
+
+#### Download metaMDBG repository  
+```
+git clone https://github.com/GaetanBenoitDev/metaMDBG.git
+```
+
+#### Create metaMDBG conda environment
+```
+cd metaMDBG
+conda env create -f conda_env.yml
+conda activate metaMDBG
+conda env config vars set CPATH=${CONDA_PREFIX}/include:${CPATH}
+conda deactivate
+
+# Activate metaMDBG environment
+conda activate metaMDBG
+
+# Compile the software
+mkdir build
+cd build
+cmake ..
+make -j 3
+```
```

Comment:
Instructions to install `metaMDBG`. These were taken directly from their GitHub page: https://github.com/GaetanBenoitDev/metaMDBG#installation.